### ISSUE TITLE
Distinguish codelists by, or otherwise identify, the source

### ISSUE THREAD

**Issue by matmaxgeds** [codeforIATI/codelist-updater#76]
CfI codelists are cool and work great, which means we are now adding codelists where the source is a single user, or a single organisation. These are often less 'official' than the first codelists added (but not less valid). I think a) it would be important for users to be able to distinguish the source or some other information more easily.

For example, the new CRS > COFOG codelist is made by (and embodies the choices of) Mark and Leigh. There might be another CRS > COFOG codelist sometime that decides some of the tricky mappings in a different way. It would be great if CfI codelists included an easy way to have both codelists hosted, with some basic information to allow users to understand the source and decide. I don't think that CRS to COFOG v2 as a different title for the second codelist would cut it.

I think the same will be true for users who are looking for a codelist replicated from (and embodying the choices made by) IATI, or directly replicated from the DAC. I could imagine that alternatives to some of these might crop up at some stage. Similarly the CRS to Humanitarian Clusters codelist.

I don't think we want to say 'official' and 'user submitted' as that often implies that one is more valid, so maybe we could just extend the source information (maybe just as easy as adding a 'creator organisation and contact field), both at the top of the page by the link to the source, and I think importantly, in the left hand side menu-bar?

I would be happy to type in the relevant data to the existing codelists if someone was able to setup the new fields?

**Comment by markbrough** [comment]
Yes, this is a good idea!! We could put this at least in the description of the codelist.

We could also maybe use the `<category/>` field for this - it appears at the moment to always say `replicated`. The steps could be something like:

1) make sure changing the category wouldn't break anything ( and check if it has any other meaning)
2) decide on agreed values for the category
3) go and change the codelists in the `Unofficial-Codelists` repo
